incubator/ibm-blockchain-network/templates/composer-pvc.yaml

```diff
@@ -0,0 +1,40 @@
+---
+{{- if not .Values.persistence.storageClass }}
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: {{ template "ibm-blockchain-composer-pvc.name" . }}
+  labels:
+    app: {{ template "ibm-blockchain-composer-pvc.name" . }}
```

Comment:
The `app` label should be the same throughout the chart.

`{{ template "ibm-blockchain-network.name" . }}`

Reply:
This is fixed.